tests/cli/test_blueprints_sanity.sh

```diff
@@ -41,5 +43,30 @@ __EOF__
         rlAssertEquals "pushed bp is found via list" "`$CLI blueprints list | grep beakerlib`" "beakerlib"
     rlPhaseEnd
 
+    rlPhaseStartTest "blueprints show"
+        rlAssertEquals "show displays blueprint in TOML" "`$CLI blueprints show beakerlib`" "`cat beakerlib.toml`"
+    rlPhaseEnd
+
+    rlPhaseStartTest "SemVer .patch version is incremented automatically"
+        # version is still 0.0.1
+        rlAssertEquals "version is 0.0.1" "`$CLI blueprints show beakerlib | grep 0.0.1`" 'version = "0.0.1"'
+        # add a new package to the existing blueprint
+        cat >> beakerlib.toml << __EOF__
+
+[[modules]]
+name = "php"
+version = "*"
+__EOF__
+        # push again
+        rlRun -t -c "$CLI blueprints push beakerlib.toml"
+        # official documentation says:
+        # if you do not change the version, the patch component is increased automatically.
```

Comment:
That's not quite right.
"If a new blueprint is uploaded with the same version the server will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is."